integrations/doubleclick-floodlight/lib/index.js

```diff
@@ -101,11 +102,27 @@ Floodlight.prototype.track = function(track) {
       each(function(variable) {
         var floodlightProp = variable.value;
         var segmentProp = variable.key.match(/{{(.*)}}/) || variable.key;
-        var segmentPropValue;
+        if (variable.key.includes('$')) {
+          segmentProp = variable.key.split('.$.');
+        }
 
-        if (Array.isArray(segmentProp)) {
+        var segmentPropValue;
+        if (Array.isArray(segmentProp) && segmentProp[0] === 'products') {
+          segmentProp = segmentProp.pop();
+          var productPropArray = [];
+          each(function(product) {
+            if (product[segmentProp]) {
+              productPropArray.push(product[segmentProp]);
+            }
+          }, track.products());
+          segmentPropValue = productPropArray;
```

Comment:
Does this need to be joined?

Reply:
In Stratconn-12 the customer requested that it be sent as a comma separated array and not a string: "DCF would expect the the u1​ parameter to be set as a comma-separated array (example:  [sku1,sku2,sku3]​)." DCF docs are pretty lacking on this topic but happy to follow up with the CSM on expected behavior before deploying.

Reply:
Got it. Sorry, I misread your comment above.